specification/scvmm/ScVmm.Management/models.tsp

```diff
@@ -32,7 +32,7 @@ model QueryForceDelete {
 }
 
 /** The provisioning state of the resource. */
-union ResourceProvisioningState {
```

Comment:
is this renaming from js handling it without flatten?

Reply:
What we propsoed here is to avoid any naming conflict within a client(either conflict with service's own namespace or with azure core namespace) and see [issue](https://github.com/Azure/typespec-azure/issues/1039). Because any conflict resolution are not peferct and may lead to wired client-side names.

So back to your question the scvmm RP has naming conflict `ResourceProvisioningState` but other langauges may not expose this issue out because of the enum item flattening. And JS follows the princeple to generate as typesec defined so we keep this name and then exposed this isue out.